boofuzz/fuzz_logger_curses.py

```diff
@@ -179,7 +179,7 @@ def log_fail(self, description="", indent_size=INDENT_SIZE):
             + (4 * indent_size + 1 - len(str(self._total_index))) * " "
             + description.strip()
         )
-        self._fail_storage.append([fail_msg, 1])
+        self._fail_storage.append([fail_msg.replace("\n", " "), curses.COLOR_RED])
```

Comment:
Ah, I see... I'll add some constants for our color pairs. We have failures and errors in this case, rather than warnings and errors. So it looks like before this PR, we were using white for failures and red for errors? White actually does make sense for errors, since they're being rendered in an error box already. I think I'll just put the colors back.

Reply:
Correct, before it was white for failures and red for errors.
Adding constants is definitely a good idea.